ci/dhis2-verify-app.yml

```diff
@@ -83,31 +83,22 @@ jobs:
     #          with:
     #              node-version: 12.x
     #
-    #        - uses: c-hive/gha-yarn-cache@v1
-    #        - run: yarn install --frozen-lockfile
-    #
-    #        - name: Install Cypress binary
-    #          run: yarn cypress install
-    #
     #        - name: End-to-End tests
     #          uses: cypress-io/github-action@v2
     #          with:
-    #              install: false
+    #              # This should be a command that serves the app without opening a browser.
+    #              # For example: `BROWSER=none d2-app-scripts start`
+    #              start: yarn start:nobrowser
```

Comment:
```suggestion
    #              start: d2-app-scripts start
```

Reply:
I went with yarn d2-app-scripts start, as I wasn't sure it would work without the yarn prefix.